docs/en/interfaces/http.md

```diff
@@ -23,11 +23,13 @@ Web UI can be accessed here: `http://localhost:8123/play`.
 ![Web UI](../images/play.png)
 
 
-In health-check scripts use `GET /ping` request. This handler always returns “Ok.” (with a line feed at the end). Available from version 18.12.13.
+In health-check scripts use `GET /ping` request. This handler always returns “Ok.” (with a line feed at the end). Available from version 18.12.13. If users want to get more sophisticated health checks, it can use the `GET /replicas_status` request.
```

Comment:
Inconsistent wording: users, it.

It is not a "more sophisticated health check", it is a check specifically for replicas status and nothing else.